tix_clock/tix_clock.ino

```diff
@@ -37,3 +37,16 @@ void SetOutputPins(int outputPins[], int numberOfPins)
         pinMode(outputPins[i], OUTPUT);
     }
 }
+
+void GiveMeRandomNumber()
+{
+    int GiveMeRandomNumber = 0;
+    int lowerLimit = 1;
+    int upperLimit = 100;
+
+    GiveMeRandomNumber = random(lowerLimit, upperLimit);
+    Serial.print("Numero random = ");
```

Comment:
Translate to english, and use: Serial.println("Random number = %d", randomNumber);

Please add the lines to setup the serial port in the void setup